src/js/hooks/cycleCount/useInventoryTransactionsTab.jsx

```diff
@@ -298,7 +310,7 @@ const useInventoryTransactionsTab = ({
         className="rt-td multiline-cell"
       >
         <div className="limit-lines-1">
-          {_.capitalize(getValue())}
+          {reasonCodes.find((c) => c.value === getValue())?.label}
```

Comment:
I don't like that we are looping over reason codes to find the appropriate code to display (and I don't like the need for fetching). Can we add an enum on the front end and then get the label, calling something like ReasonCodes[getValue()]?